packages/components/src/components/Snackbar/SnackbarProvider.tsx

```diff
@@ -24,10 +28,18 @@ export const SnackbarProvider: React.FC<{ children: ReactNode }> = ({
         duration={1000000000000} // Essentially indefinite until dismissed
         offset={offset}
         placement="bottom"
-        renderToast={(toast) => <Snackbar {...toast as SnackbarProps} />}
+        renderToast={(toast) => <Snackbar {...(toast as SnackbarProps)} />}
```

Comment:
```suggestion
        renderToast={(toast) => <Snackbar {...toast as SnackbarProps} />}
```
?

Reply:
Autoformatter forces this for some reason. I can save without formatting